<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>909</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font-family:&quot;Palatino Linotype.&quot;;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_superior">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#668;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../imagenes/graduado (4).png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Palatino Linotype.</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>REGISTRATE YA! 
NO ESPERES MAS...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_inferior">
      <property name="styleSheet">
       <string notr="true">#frame_inferior{
	background:#ffffff
	
}
.QLabel{
color:#000;

}

.QLineEdit{
	background-color: #DED9FA;
	border:none;
	padding:4px;
	border-bottom:1.5px solid #111;
	border-radius:4px;
	
}

.QLineEdit:focus{
	border-bottom:2px solid #9813FC;
}

.QPushButton{
	background:#915EF6;
	padding:6px;
	border:none;
	border:1px solid #333;
	border-radius:4px
}

.QPushButton:hover{
	background:#667;
	
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>true</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(170, 85, 255);</string>
            </property>
            <property name="text">
             <string>Datos del Empleado:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Nombres:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtNombres">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Apelldios</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtApellidos">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="label_11">
               <property name="font">
                <font>
                 <family>Palatino Linotype.</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>DNI:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtDni">
               <property name="font">
                <font>
                 <family>Palatino Linotype.</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QLabel" name="label_10">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Palatino Linotype.</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Elija un cargo:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cboCargo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Palatino Linotype.</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>Profesor</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Administradora</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Directora</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Asesor</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(170, 85, 255);</string>
            </property>
            <property name="text">
             <string>Creacion de Cuenta:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Palatino Linotype.</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Crear usuario:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtUsuario">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Crear contraseña:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtPassword">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Confirmar contraseña:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtConfirmarCon">
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnRegistrar">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Crear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConfirmar">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Palatino Linotype.</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Confirmar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
